src/ni/datastore/data/_types/_published_measurement.py

```diff
@@ -19,7 +19,13 @@
 
 
 class PublishedMeasurement:
-    """Information about a measurement published to the data store."""
+    """Represents a measurement that has been published to the data store.
+
+    A published measurement contains the moniker of the published measurement
```

Comment:
I don't want to put too much emphasis on the moniker. It's an implementation detail on how we store the measurement data. Our API allows users to get data based on the published measurement/condition, so you could write your whole test application without ever touching a moniker.

I would put more emphasis on the PublishedMeasurement containing measurement data and associated metadata.